runtime/doc/map.txt

```diff
@@ -889,6 +889,46 @@ or `unnamedplus`.
 The `mode()` function will return the state as it will be after applying the
 operator.
 
+						*modifyOtherKeys* *csi-u*
```

Comment:
- this probably belongs in term.txt.
- Using the `tui-` prefix also helps user discovery.


```suggestion
						*tui-modifyOtherKeys* *tui-csiu*
```